cmd/kubebuilder/create/resource/controller.go

```diff
@@ -42,11 +42,17 @@ import (
     "github.com/kubernetes-sigs/kubebuilder/pkg/controller"
     "github.com/kubernetes-sigs/kubebuilder/pkg/controller/types"
     "k8s.io/client-go/tools/record"
-
+    {{if .CoreType}}
```

Comment:
What does CoreType means here ?

Reply:
Should be `ControllerOnly`, will fix it.

Reply:
Done

Reply:
I changed it back to `CoreType` in `create controller` command. It makes sense now compared with `ControllerOnly`.